transcript_tools/style_guide.md

```diff
@@ -0,0 +1,132 @@
+
+# Transcriptions Style Guide
+
+This is a set of guidelines, that will help maintain a consistent style across
+transcriptions done by multiple people.  Many are arbitrary, and possibly wrong.
+Feel free to suggest changes or corrections.
+
+## File formatting
+
+Use standard markdown syntax as needed.  Titles probably aren't appropriate.
+
+Before submitting a PR with a new, transcript, word-wrap the text to 80
+characters.
+
+## Names
+
+Names should be shown in __Bold__.
+
+The first time a speaker speaks, write their full name.  After that, each time
+the speaker changes, write just the first name of the new speaker.  If they go
+on for multiple paragraphs, only put their name at the start of the first
+paragraph.
+
+If a speaker mentions the name of another person, project, group, website, etc.,
+try to make sure the spelling is correct.  If it's not possible to determine
+what they're referring to, leave a (?) marker near the name.
+
+## Editorial decisions
+
+None of these have obviously correct answers.  Editorial judgment is called for.
+Just do the best you can.
+
+Many words and sounds can be omitted in the transcript:
+- Filler words: "The answer is, uh, hm, six."
+- False starts: "The answer should... the answer is six."
+- Repeated words: "The answer is, is six."
+All of these can be written "The answer is six."
+
+Often the word "like" can be a filler word: "The, like, answer, like, is six."
+It probably improves readability to remove it.  Be careful, though— sometimes
+"like" is part of the meaning of the sentence, e.g. "They were like, this is
+broken." (Where "like" indicates a thought or statement attributed to someone
+else.)
+
+How much to trim is ultimately a judgment call.  Try to leave the speaker's
+style in place but also make the sentences readable.
+
+Try to insert sentence breaks where needed; one sentence that goes on for too
+long hurts readability.  If that means you need to start sentences with "And",
+"But", "So", etc., that's still better than run-on sentences that go on forever.
+
+If a speaker cuts themselves off mid-sentence, and jumps topics, or interrupts
+or corrects themselves, use the UTF-8 em-dash character "—" to mark the break.
+Example: "As you would expect, you can compile— the Rust compiler will use the
+usual algorithms."
+
+Also use an em-dash "—" to mark the point when a speaker stops suddenly (and
+doesn't resume) or if a speaker is interrupted by someone else.
+
+Don't try to fix sentences that are not grammatically correct.  If that's what
+the speaker said, it's what should go in the transcript.  If it's not clear
+whether they used the correct word (e.g. you can't tell whether they used the
+right verb tense), by all means use the correct one by default.
+
+## Hyperlinks
+
+So far the transcripts have not contained many hyperlinks.  This hasn't been
+discussed among contributors yet.
+
+What do you think the link policy should be?
```

Comment:
I think links should generally be avoided, and instead inserted into the show notes if they are missing from there. The exception might be if it is not at all clear from the sentence what is going on (such as if the name of a feature is given incorrectly).

Reply:
I agree.

Reply:
I pasted your answer into the document.